camviewer.pyw

```diff
@@ -55,7 +55,7 @@ if __name__ == '__main__':
   except:
     pass
 
-  QApplication.setGraphicsSystem("raster")
+  #MCB??? QApplication.setGraphicsSystem("raster")
```

Comment:
I was curious:

From https://doc.qt.io/qt-5/sourcebreaks.html
> QApplication::setGraphicsSystem() is removed, because the introduction of [QPA](https://doc.qt.io/qt-5/qpa.html) made it redundant.

```suggestion
```

Reply:
I wasn't sure if it was needed when I couldn't find it, so I temporarily commented it out.  Now that we're sure it's not needed, it's out.